### ISSUE TITLE
Warning about "unique key" prop when using with NextJS

### ISSUE THREAD

**Issue by sylvix** [garronej/tss-react#120]
Hello! Started using your library since I liked "makeStyles->useStyles" in MaterialUI v4 very much. Trying to use it as a standalone and with NextJS. After having set up everything it works but shows this warning into the console:
```
Warning: Each child in a list should have a unique "key" prop. See https://reactjs.org/link/warning-keys for more information.
    at style
    at Head (webpack-internal:///./node_modules/next/dist/pages/_document.js:246:1)
    at html
    at Html (webpack-internal:///./node_modules/next/dist/pages/_document.js:645:106)
    at Document (webpack-internal:///./node_modules/next/dist/pages/_document.js:14:1)
```
The problem is here: **src/next.tsx:68**
Instead of 
```
<style id={insertionPointId} />,
```
It should be:
```
<style id={insertionPointId} key="insertion-point" />, // or some other meaningful unique key
```
Since it's an array of React elements, every one of them should have a key.

**Comment by garronej** [comment]
Damn sorry!  
Releasing a fix right now!  
Thanks for reporting, I just updated the Next.js utility

**Comment by garronej** [comment]
It's fixed in 4.3.4 thanks for reporting!

**Comment by sylvix** [comment]
Thank you! That was quick!
